docs/integrations/sources/google-ads-migrations.md

```diff
@@ -1,5 +1,24 @@
 # Google Ads Migration Guide
 
+## Upgrading to 3.0.0
+
+This release upgrades the Google Ads API to version 15 which causes the following changes in the schemas:
```

Comment:
This release upgrades the Google Ads API from Version 13 to Version 15 which causes the following changes in the schemas:

Reply:
Updated